packages/frontend/src/components/message/MessageListAndComposer.tsx

```diff
@@ -175,9 +175,12 @@ export default function MessageListAndComposer({ accountId, chat }: Props) {
 
         for (const file of sanitizedFileList) {
           const path = await writeTempFileFromFile(file)
+          const msgViewType: Viewtype = file.type.startsWith('image')
+            ? 'Image'
+            : 'File'
```

Comment:
I wonder if we should keep improving this. E.g. add support for GIFs, webxdcs, etc.

Reply:
maybe with the composer refactor, currently core changes stuff anyways. I don't have the full current behaviour loaded to my mind, I'm fine with it at the moment as long as it works.